faqs.html

```diff
@@ -13,6 +13,18 @@ <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
       <div class="collapsible-header black clemson-orange-text">When is the event?</div>
       <div class="collapsible-body"><span>CUCTF is taking place October 3rd, 2020.</span></div>
     </li>
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <div class="card-content"> <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p>CU Cyber is aware of the lingering uncertainty with the COVID-19 pandemic. Our hearts are with those affected by the virus, and our leadership team is taking the steps necessary to ensure CUCTF 2020 is up and operational come October 3rd. </p>
+        <p>At this time, Clemson University has not released a statement regarding whether or not Fall Semester will operate under normal conditions. If Clemson University does not permit this event to take place, CUCTF 2020 will become a virtual event that will allow universities to participate in the event remotely. We will implement this plan by using the following strategy: </p>
+                                        <ul class="browser-default">
+                                          <li>Present opening and closing ceremonies via Zoom </li>
+                                          <li>Utilize Slack/Discord to maintain a connection with participants</li>
+                                          <li>Increase CTF infrastructure to allow more universities to participate online</li>
+                                        </ul></div></div>
+    </li>
```

Comment:
Fix a few errant whitespaces, clarify we will use Slack xor Discord, reword increase infrastructure, and fix HTML `</div>` which would not match any opening `<div>` tag with the above suggestion.

```suggestion
        <ul class="browser-default">
          <li>Present opening and closing ceremonies via Zoom</li>
          <li>Utilize Slack or Discord to maintain a connection with participants</li>
          <li>Build out larger CTF infrastructure to allow more universities to participate online</li>
        </ul>
      </div>
```